src/common/BaseCommand.ts

```diff
@@ -82,10 +87,9 @@ export abstract class BaseCommand
             // If so then we override the default parser with our parser which runs
             // the validation check and only accepts the flag value if validation passes.
             const configEntries = Object.entries(flag[1]);
-            const hasValidate = configEntries.find(
-                (keyValuePair) => keyValuePair[0] === 'validate'
-            );
+            const hasValidate = configEntries.find((keyValuePair) => keyValuePair[0] === 'validate');
             if (hasValidate) {
+                // eslint-disable-next-line @typescript-eslint/unbound-method, no-param-reassign
```

Comment:
nit: looks like one-off one-liner. not sure if you would prefer every eslint disable up at the top of the file.

Reply:
Yes this is intentional. I actually prefer one-off disables over blanket file-level disables. The reason I left the file-level disables in other places is b/c there were several instances of those rule violations per file. As I mentioned [here](https://github.com/forcedotcom/lwc-dev-mobile-core/pull/83/files#r1598781315), going from `commonjs` to `node16` and also extending our tsconfig & eslint rules from `sf*` rules means stricter rule enforcement. Rather than boiling the whole ocean in this PR (which is already very large) I've opted for disabling some of these rules per file. We can always revisit in another PR and try to enable as many of them as we can.